bitnami/argo-cd/crds/extension.yaml

```diff
@@ -1,95 +0,0 @@
-# CRDs source: https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/templates/crds/crd-extension.yaml (2.6.6)

```

Comment:
The `argocdextensions.argoproj.io` is no longer included in the upstream CRDs:
https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/templates/crds/crd-extension.yaml

See https://github.com/argoproj/argo-helm/pull/2406for more information